src/utils/threshold.ts

```diff
@@ -69,7 +69,10 @@ export function extractStatusCounts(profile: ContextualizedProfile, severity?: s
   for (const c of profile.contains.filter(control => control.extendedBy.length === 0)) {
     const control = c.root
     const status: ControlStatus = control.hdf.status
-    const controlSeverity: Severity = control.hdf.severity
+
+    // using impact here rather than control.hdf.severity is more reflective of how important these controls are
```

Comment:
instead of just trying to make the old tests pass, please validate that using impact instead of severity here leads to correct/accurate counting

Reply:
I have discussed this with @aaronlippold and he said that the intention is to count severities, so I reverted my change. However, there were still issues with the code's assumption that every control with a status of N/A has a severity of none, which is now not the case. I have expanded the status counting function to be able to count N/A controls that do have non-none (low, medium, high, critical) severities. The way this changes the functionality of the SAF CLI validation function is that it may have lower `no_impact.none` counts and higher `no_impact.low`, `no_impact.medium`, etc. counts. The higher counts won't have any impact because current validation counts files don't have any set thresholds for them because they never existed before. The only way that this can change current pipelines which use this is if their threshold files, for some reason, specify a minimum amount of N/A controls with no severity. This change might divert some of those "N/A + none" to other N/A buckets.